code/game/objects/items/devices/flashlight.dm

```diff
@@ -452,6 +452,15 @@
 	icon_state = "glowstick_yellow"
 	item_state = "glowstick_yellow"
 
+/obj/item/device/flashlight/glowstick/bluespace

```

Comment:
This will need to go in a _vr file since flashlight.dm is a polaris file.

Reply:
done